### DISCUSSION TITLE
nonfunctional :delm!  [ windows 10 ]

### DISCUSSION THREAD

**Discussion by sourcevault** [NeoVintageous/NeoVintageous#945] [General]
I am trying to delete marks using `:delm!`, but it does not seem to work.

https://github.com/NeoVintageous/NeoVintageous/assets/20347647/cc4d22f5-3f9a-43f6-9d0b-84890a7e1b98

**Comment by gerardroche** [comment]
The delmarks ex command is not implemented yet, https://github.com/NeoVintageous/NeoVintageous/issues/948.

For some reason though `:delm` is matching `:delete` which requires an argument which is the weird error message you see. Usually it would just print "Command not implemented".
